### ISSUE TITLE
Yellow lightbulb always asking to add include path

### ISSUE THREAD

**Issue by hmenzagh** [Microsoft/vscode#19772]
- VSCode Version: Latest, 1.9.0
- OS Version: OSX 10.12

Hi, I just noticed (maybe it's recent) that every time I type a new line in a .c file on vscode (or even when I clic on a function name), there is the yellow lightbulb that appears and that asks me to add an include path to the settings. But possible includes specified .. I can't get rid of it and it's driving me crazy.

Here is my c_cpp_properties.json :
```

{
    "configurations": [
        {
            "name": "Mac",
            "includePath": ["/usr/include", 
            "/Users/XXX/XXX/XXX/includes",
             "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk/usr/include"],
            "browse" : {
                "limitSymbolsToIncludedHeaders" : false,
                "databaseFilename" : ""
            }
        },
        {
            "name": "Linux",
            "includePath": ["/usr/include"],
            "browse" : {
                "limitSymbolsToIncludedHeaders" : true,
                "databaseFilename" : ""
            }
        },
        {
            "name": "Win32",
            "includePath": ["c:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/include"],
            "browse" : {
                "limitSymbolsToIncludedHeaders" : true,
                "databaseFilename" : ""
            }
        }
    ]
}
```

**Comment by Tyriar** [comment]
Do you have any extensions installed?

**Comment by bpasero** [comment]
Reading your description it seems that the issue originates from an extension and not VS Code itself. Thereby I will close this issue and kindly ask you to report this issue against the extension repository. 

Most extensions are on GitHub as well so it should be easy to file it against the extension. If you open an extension on the market place, click on the **Support** link to find out more.

<img width="196" alt="screen shot 2016-05-05 at 12 51 17" src="https://cloud.githubusercontent.com/assets/900690/15041943/263e2282-12c0-11e6-9a33-d2b3b500d61b.png">

Thanks again for your feedback!

**Comment by hmenzagh** [comment]
Indeed the issue goes away when I disable the C/C++ extension

**Comment by sean-mcmanus** [comment]
@hmenzagh It's a bug with the cpptools extension (on VS Code 1.9 or later). It should be fixed with version 0.10.1 we're releasing today.
